pages/content/posts/w3c-docs-cg.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "Florian Scholz"
+title: "Launching the W3C Docs Community Group"
+date: "2025-04-17"
+tags: ["announcements"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "We're launching a new W3C Community Group, the Docs CG."
+---
+
+We're launching a new W3C Community Group, the [Documentation CG](https://www.w3.org/community/docs-cg/) (Docs CG). This group is an open community with the mission to ensure web developers and designers worldwide have the best information available so they can build things on the web platform.
+
+The group is open to anyone to join. If you would like to join, please visit the [group's homepage](https://www.w3.org/community/docs-cg/join). To follow along with the group's activities, you can watch the [repository](https://github.com/w3c-cg/webdocs) on GitHub where meeting minutes will be posted and where asynchronous discussions will happen.
+
+The group is meant for technical writers, documentation practitioners, specification authors, subject matter experts, and web developers to collaborate. While we, at Open Web Docs, have a lot experience with MDN Web Docs, we also want to explicitly invite other web platform documention website authors to the group to share best practices and create a common understanding of how best to document web platform technologies.
+
+We believe that technical writing is essential for web platform technologies to be adopted and understood globally. We further believe it is necessary to have more conversations between:
+
+- specification authors, who design web technologies,
+- browser implementers, who implement these technologies,
+- technical writers, who document them, and
+- web developers and designers, who rely on high quality documentation to use web technologies.
+
+By offering an active community that brings all of these perspectives together, the Docs CG aspires to help write better web platform documentation for all.
+
+And also, by existing within W3C Community Groups program, the Docs CG will greatly benefit from being part of a wider community which will allow us to coordinate our efforts in collaboration with other (horizontal) W3C groups, such as [WebDX](https://www.w3.org/community/webdx/), [Security](https://www.w3.org/mission/security/), [Privacy](https://www.w3.org/mission/privacy/), [Accessibility](https://www.w3.org/mission/accessibility/), [Internationalization](https://www.w3.org/mission/internationalization/), [Sustainability](https://www.w3.org/community/sustyweb/), and more.
+
+The W3C Docs CG will be chaired by Lola Odelola ([lolaslab.co](https://lolaslab.co)) and Florian Scholz (Open Web Docs). We'd love to welcome you to our [slack channel](https://w3ccommunity.slack.com/archives/C08KNG3GFEF) ([invite](https://www.w3.org/slack-w3ccommunity-invite)) and to our upcoming [Docs CG calls](https://www.w3.org/groups/cg/docs-cg/calendar/)!
```

Comment:
```suggestion
The W3C Docs CG will be chaired by Lola Odelola ([lolaslab.co](https://lolaslab.co)) and Florian Scholz (Open Web Docs). We'd love to welcome you to our [Slack channel](https://w3ccommunity.slack.com/archives/C08KNG3GFEF) ([invite](https://www.w3.org/slack-w3ccommunity-invite)) and to our upcoming [Docs CG calls](https://www.w3.org/groups/cg/docs-cg/calendar/). See you Tuesday for the [Docs CG kick off!](https://www.w3.org/events/meetings/5db757dd-664b-4b78-a962-bd14e713c899/)
```

Reply:
Adding a date may be unideal depending on:
1. When this post goes out
2. If someone will come back to update the post to remove the date.

Reply:
Let's not have the Tuesday kickoff call in there. I reckon Tuesday will be mostly for us to get organized, not a big deal if there isn't a full house yet.

```suggestion
The W3C Docs CG will be chaired by Lola Odelola ([lolaslab.co](https://lolaslab.co)) and Florian Scholz (Open Web Docs). We'd love to welcome you to our [Slack channel](https://w3ccommunity.slack.com/archives/C08KNG3GFEF) ([invite](https://www.w3.org/slack-w3ccommunity-invite)) and to our upcoming [Docs CG calls](https://www.w3.org/groups/cg/docs-cg/calendar/).
```